docs/data_management/lts/tutorial/lts_tutorial.md

```diff
@@ -0,0 +1,119 @@
+
+First, access our interactive Open OnDemand (OOD) portal at https://rc.uab.edu and create an [interactive HPC Desktop session](../../../cheaha/open_ondemand/hpc_desktop.md). Once the HPC Desktop session lunched, open the terminal and run the below commands.
+
+```bash
+module load Anaconda3
+conda create -n s3 
+conda activate s3
+pip install s3cmd
+```
+
+### Install s3cmd on Your Local Systems
+
+To install s3cmd on your local machine, please follow the instructions provided in [our s3cmd documentation for local installation](../../../data_management/lts/interfaces.md/#installation-of-s3cmd-and-s5cmd-on-personal-system).
+
+### Configuring s3cmd for LTS Buckets
+
+Properly configuring s3cmd is important for working with LTS buckets. The steps vary depending on whether you have a single LTS account or multiple accounts to manage. Follow the appropriate guide below for your setup.
+
+#### Single LTS Account (Individual Users and Researchers)
+
+For individual users with a single LTS account, follow these steps:
+
+Open a terminal through the HPC Desktop, activate the `s3` environment created in the [Install s3cmd using within Anaconda Environment](./lts_tutorial.md/#install-s3cmd-using-pip-within-anaconda) section, and run the below command:
+
+```bash
+s3cmd --configure
+```
+
+This will prompt you for your access key and secret access key associated with your LTS account. Enter these details carefully. You will be asked for additional information, see examples [here](../interfaces.md/#configuring-s3cmd). Once completed, `s3cmd` will create a default configuration file named `.s3cfg` in your home directory (`$HOME`).
+
+#### Multiple LTS Account (PIs/Cores and Data managers)
+
+If you are a Lab PI or Core and manage both a personal and a lab/core LTS account, you need separate profiles for each. Follow these steps: 
+
+- **To create a profile for your personal LTS Account**: Run `s3cmd --configure -c $HOME/personal_profile`. Replace `personal_profile` with your preferred profile name. Enter your personal LTS access credentials when prompted. This will create a configuration file named  `personal_profile` in your home directory (`$HOME`).
+- **To create a profile for your Lab/Core LTS Account**: Run the command, `s3cmd --configure -c $HOME/lab_core_profile`. Replace `lab_core_profile` with your preferred profile name. Enter your lab/core LTS access credentials when prompted. This will also create a configuration file named  `lab_core_profile` in your home directory (`$HOME`).
+
+In both cases, you will be asked for additional information beyond your access key and secret access key. For examples and detailed guidance, see [here](../interfaces.md/#configuring-s3cmd).
+
+### How to I switch between `s3cmd` configuration profiles?
```

Comment:
Let's move this to an FAQ section at the end.

Reply:
This part is omitted from this tutorial  for now, planning to cover it in a separate guide with multiple LTS accounts.